doc/classes/CanvasItem.xml

```diff
@@ -404,7 +404,7 @@
 		<method name="get_canvas_transform" qualifiers="const">
 			<return type="Transform2D" />
 			<description>
-				Returns the transform matrix of this item's canvas.
+				Returns the transform from the coordinate system of the canvas, this item is in to the [Viewport]s coordinate system.
```

Comment:
"is in to" seems to be a typo (also occurs below).

Reply:
Or maybe not, but the sentence syntax is confusing.

Adding a missing comma helps, if I understand the intent correctly:
```suggestion
				Returns the transform from the coordinate system of the canvas, this item is in, to the [Viewport]s coordinate system.
```

I would suggest rephrasing as it's pretty difficult to make sense of IMO. I can help formulate but I'm not sure what the intended meaning actually is.

Reply:
Thanks for the offer. As explanation: Transforms are meant to translate between two coordinate systems. In this case
- from the coordinate system of the canvas (CanvasLayer or default canvas of the Viewport), that this CanvasItem is in
- to the coordinate system of the Viewport, that this CanvasItem is in.
You are right, that a comma would be better for clarity. Will update the PR later and also think about how to word this more clearly.